.travis.yml

```diff
@@ -37,15 +37,15 @@ matrix:
     - os: linux
       env: PROJECT_DIR=examples/double-conversion TOOLCHAIN=android-ndk-r15c-api-21-armeabi-v7a-neon-clang-libcxx
     - os: linux
-      env: PROJECT_DIR=examples/double-conversion TOOLCHAIN=analyze
```

Comment:
Please keep the order, in future merge from `pkg.template` will happen. It will simplify the understanding of changes and resolving conflicts.

Reply:
OK thanks.